extensions/duplicates/PhylumExt.toml

```diff
@@ -0,0 +1,5 @@
+name = "duplicates"
+entry_point = "main.ts"
+
+# [permissions]
+# write = ["./duplicates.txt"]
```

Comment:
Do we plan on enabling/using these permissions in the future? If we do, then I'd say it's fine to keep this.

Reply:
If you look at the example, you'll see that the commented-out lines here are variations to the example introduced at the end of the documentation to introduce the concept of permissions.

The goal is to provide all the necessary code to ensure users can trace back their problems if they fail to follow along.